z_infra_provisioning/cloud_infra_center/ocp_upi/README.md

```diff
@@ -348,6 +433,8 @@ The playbook places the Control Plane in a Server Group with "soft anti-affinity
 
 The master nodes should load the initial Ignition and then keep waiting until the bootstrap node stands up the Machine Config Server which will provide the rest of the configuration.
 
+***Note***: This playbook will fail with `UserWarning: You are specifying a cacert for the cloud envvars:RegionOne but also to ignore the host verification`. You can monitor this issue [#72](https://github.com/IBM/z_ansible_collections_samples/issues/72) to get latest status.
```

Comment:
It will not always fail(it will success in z/M env), so I think we can say "This playbook may fail..."

Reply:
Update it. Thx.